dtu-hpc-python3/setup-python3.sh

```diff
@@ -49,6 +49,12 @@ module load boost
 export PATH="$HOME/bin:$PATH"
 export LD_LIBRARY_PATH=$HOME/lib:$LD_LIBRARY_PATH
 
+# NOTE: Not sure if this should be added
```

Comment:
`module load cuda` will automatically modify the `LD_LIBRARY_PATH`. `CUDA_HOME` is not a magic env variable, I believe they just use it to make the setup guide more readable.